physicalplan/unary/unary.go

```diff
@@ -27,16 +30,22 @@ type unaryNegation struct {
 func NewUnaryNegation(
 	next model.VectorOperator,
 	stepsBatch int,
+	op parser.ItemType,
 ) (model.VectorOperator, error) {
 	u := &unaryNegation{
 		next:       next,
 		stepsBatch: stepsBatch,
+		op:         op,
 	}
 
 	u.workers = worker.NewGroup(stepsBatch, u.workerTask)
 	return u, nil
 }
 
+func (o *unaryNegation) Explain() (me string, next []model.VectorOperator) {
```

Comment:
Interesting that this wasn't added. Should we have a separate `Explainable` interface and make `Explain` return 
```
(me string, next []Explainable) 
```
so that we don't miss such cases in the future?

We can also add the method on `model.VectorOperator`, but having a separate interface will help with reusing the same mechanism for visualizing the logical plan.

cc @bwplotka

Reply:
👍🏽